src/model/enhanceCAPI.ts

```diff
@@ -0,0 +1,84 @@
+import { validateAsCAPIType } from '@root/src/model/validate';
+import { addDividers } from '@root/src/model/add-dividers';
+import { enhanceDots } from '@root/src/model/add-dots';
+import { setIsDev } from '@root/src/model/set-is-dev';
+import { enhanceImages } from '@root/src/model/enhance-images';
+import { enhanceInteractiveContentElements } from '@root/src/model/enhance-interactive-content-elements';
+import { enhanceNumberedLists } from '@root/src/model/enhance-numbered-lists';
+import { enhanceInteractiveAtomElements } from '@root/src/model/enhance-interactive-atom-elements';
+import { enhanceBlockquotes } from '@root/src/model/enhance-blockquotes';
+import { enhanceEmbeds } from '@root/src/model/enhance-embeds';
+
+class CAPIEnhancer {
```

Comment:
`CAPIEnhancer` is large enought if feels ok to split it out of `src/web/server/render.ts`